src/file/exceptions.php

```diff
@@ -12,23 +12,20 @@
 
 use namespace HH\Lib\Experimental\{IO, OS};
 
-abstract class Exception
+/**
+ * An exception thrown when opening a file fails.
+ */
+final class OpenException
```

Comment:
`OpenException` seems weird and the name isn't really as helpful as `LockAcquisitionException` or `HostResolutionException`